spec/hosts/nodes/auxtel-hcu01.cp.lsst.org_spec.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'auxtel-hcu01.cp.lsst.org', :sitepp do
+  on_supported_os.each do |os, os_facts|
+    next unless os =~ %r{almalinux-9-x86_64}
+
+    context "on #{os}" do
+      let(:facts) do
+        lsst_override_facts(os_facts,
+                            is_virtual: false,
+                            virtual: 'physical',
+                            dmi: {
+                              'product' => {
+                                'name' => 'UNO-1483G-434AE',
+                              },
+                            })
+      end
+      let(:node_params) do
+        {
+          role: 'atshcu',
+          cluster: 'auxtel-ccs',
+          site: 'cp',
+        }
+      end
+
+      it { is_expected.to compile.with_all_deps }
+
+      include_examples 'baremetal no bmc'
+      include_context 'with nm interface'
+
+      it { is_expected.to have_nm__connection_resource_count(4) }
+
+      context 'with eno1' do
+        let(:interface) { 'eno1' }
+
+        it_behaves_like 'nm enabled interface'
+        it_behaves_like 'nm ethernet interface'
+        it_behaves_like 'nm dhcp interface'
+      end
+
+      context 'with enp6s0' do
+        let(:interface) { 'enp6s0' }
+
+        it_behaves_like 'nm enabled interface'
+        it_behaves_like 'nm ethernet interface'
+        it_behaves_like 'nm manual interface'
+        it { expect(nm_keyfile['ipv4']['address1']).to eq('192.168.1.1/24') }
+      end
+
+      context 'with enp7s0' do
+        let(:interface) { 'enp7s0' }
+
+        it_behaves_like 'nm disabled interface'
+        it_behaves_like 'nm ethernet interface'
+        it { expect(nm_keyfile['ipv6']['method']).to eq('disabled') }
+      end
+
+      context 'with enp8s0' do
```

Comment:
Two disabled interfaces listed out is OK but if it was any more I'd have preferred to see an array iterated over to keep it DRY.